src/table/cellRenderers/cellContentRenderers.tsx

```diff
@@ -0,0 +1,26 @@
+export const roundNumber = (
+  num: number,
+  sf: number,
+  scientificNotation: boolean
+): string => {
+  /*
+  In normal mode, do not round to the left of the decimal point,
+  even if the number of significant figures says that you should.
+  For example, 916.3 with a specified precision of 2 sig figs should be
+  displayed as 9.2e2 or 916 in scientific or normal mode respectively.
+  */
+
+  // TODO: what to return here?
+  if (sf <= 0) return 'Invalid significant figure';
```

Comment:
Personally not a fan. As long as we have data to display, I say we do whatever we can to display it. If the rules by which they want us to format that data aren't valid (in this case, an invalid number of significant figures), just return it back to them as is. Therefore, if the input is 916.3, return 9.2e2 or 916 in scientific or normal mode respectively. Just my two cents.

Reply:
Do you mean if we get 916.3 and the significant figures is invalid we return 916.3 (or 9.163e2)? It's hard for us to return 916 or 9.2e2 as those both have implied what the significant figures are (1/2/3 or 2 respectively)

Reply:
Maybe I'm misunderstanding. What sort of scenario would lead to significant figures being <= 0?

Reply:
I wouldn't really expect it to happen (if the metadata is good!) - I just put in the condition as otherwise the function throws an exception if it gets something <= 0

Reply:
Right that's what I thought. What I'm trying to say is that, if significant figures is <= 0, the function can't process it, it's invalid. So just return its input.

Reply:
yeah that's an easy change :) I think I misread the github notification and read the part of the code comment saying `9.2e2 or 916` as being part of your comment 😅

Reply:
Damn it, that was my bad! I realised I'd mistyped what I wanted to change so quickly tried to edit it but you beat me to it and saw the original comment. Hope my meaning is now clear :)